javalib/src/main/scala/java/util/AbstractSequentialList.scala

```diff
@@ -0,0 +1,42 @@
+package java.util
+
+abstract class AbstractSequentialList[E] protected ()
+    extends AbstractList[E] {
+
+  def get(index: Int): E =
+    listIterator(index).next()
```

Comment:
```
val iter = listIterator(index)
if (iter.hasNext) iter.next()
else throw new IndexOutOfBounds(index.toString)
```
should throw all the `IndexOutOfBounds` without using potentially linear additional checks like `index != size`.